executor/api/test/seldonmessage_test_client.go

```diff
@@ -2,20 +2,18 @@ package test
 
 import (
 	"context"
-	"github.com/seldonio/seldon-core/executor/api/client"
 	"github.com/seldonio/seldon-core/executor/api/grpc/seldon/proto"
 	"github.com/seldonio/seldon-core/executor/api/payload"
 	"github.com/seldonio/seldon-core/operator/apis/machinelearning/v1"
 	"io"
 	"net/http"
-	"testing"
 )
 
 type SeldonMessageTestClient struct {
-	t           *testing.T
-	chosenRoute int
-	errMethod   *v1.PredictiveUnitMethod
-	err         error
+	ChosenRoute int
+	ErrMethod   *v1.PredictiveUnitMethod
+	Err         error
+	ErrPayload  payload.SeldonPayload
```

Comment:
Do we need the `ErrPayload` field? It doesn't seem to be used on the PR.

Reply:
updated a test to use it